src/webgl.js

```diff
@@ -278,8 +274,8 @@ function init_conv1_1_program(gl) {
     out1 = vec4(0.0, 0.0, 0.0, 0.0);
     out2 = vec4(0.0, 0.0, 0.0, 0.0);
     out3 = vec4(0.0, 0.0, 0.0, 0.0);
-    float outX = float(gl_FragCoord.x);
-    float outY = float(gl_FragCoord.y);
+    float outX = gl_FragCoord.x;
+    float outY = gl_FragCoord.y;
```

Comment:
Given this, you can now do `vec2 out = gl_FragCoord;`

Reply:
So these "out" vars, outX and outY, are not actually out vars. But we don't need them since they are simply the gl_FragCoord as you pointed out. I've removed them.